core/src/components/datetime/datetime.tsx

```diff
@@ -197,6 +197,8 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() presentation: 'date-time' | 'time-date' | 'date' | 'time' | 'month' | 'year' | 'month-year' = 'date-time';
 
+  private prevPresentation: string | null = null;
```

Comment:
~ Doesn't look like we have a strong stance on this, but organizing private variables at the top of the class is a useful practice. If we moved this, we should probably move `todayParts` as well.